tailwind.config.js

```diff
@@ -22,11 +22,11 @@ module.exports = {
       "h3-tall": ["24px", "30px"],
     },
     screens: {
-      xxs: "768px",
-      xs: "768px",
-      sm: "768px",
-      md: "768px",
-      lg: "992px",
+      xxs: "280px",
+      xs: "325px",
+      sm: "450px",
+      md: "550px",
+      lg: "768px",
       xl: "992px",
       xxl: "1170px",
     },
```

Comment:
So the problem with this is that the rest of the site (the header + footer) gets shifted into the middle.

| tablet | mobile |
|--------|-------|
|  <img width="874" alt="Screen Shot 2021-05-30 at 12 33 38" src="https://user-images.githubusercontent.com/2454380/120112545-4d139d00-c144-11eb-8ed1-7cbb6eb37031.png">    |   
<img width="656" alt="Screen Shot 2021-05-30 at 12 33 47" src="https://user-images.githubusercontent.com/2454380/120112547-4dac3380-c144-11eb-9572-21e42ed409c4.png">  |

I think it's actually much simpler what would be a better fix here: we should set a max-width on the div that contains all the topic boxes (cap it somewhere between 400-500px) and remove the left margin (otherwise, all the content is centred. That way, we get something that looks more like this.

Reply:
Fixed this as per our conversation.